src/BlockTypes/MiniCart.php

```diff
@@ -421,43 +414,23 @@ protected function get_markup( $attributes ) {
 			return '';
 		}
 
-		$cart                = $this->get_cart_instance();
-		$cart_contents_count = $cart->get_cart_contents_count();
-		$cart_contents_total = $cart->get_subtotal();
-
-		if ( $cart->display_prices_including_tax() ) {
-			$cart_contents_total += $cart->get_subtotal_tax();
-		}
-
 		$classes_styles  = StyleAttributesUtils::get_classes_and_styles_by_attributes( $attributes, array( 'text_color', 'background_color', 'font_size', 'font_weight', 'font_family' ) );
 		$wrapper_classes = sprintf( 'wc-block-mini-cart wp-block-woocommerce-mini-cart %s', $classes_styles['classes'] );
 		if ( ! empty( $attributes['className'] ) ) {
 			$wrapper_classes .= ' ' . $attributes['className'];
 		}
 		$wrapper_styles = $classes_styles['styles'];
 
-		$aria_label = sprintf(
-			/* translators: %1$d is the number of products in the cart. %2$s is the cart total */
-			_n(
-				'%1$d item in cart, total price of %2$s',
-				'%1$d items in cart, total price of %2$s',
-				$cart_contents_count,
-				'woo-gutenberg-products-block'
-			),
-			$cart_contents_count,
-			wp_strip_all_tags( wc_price( $cart_contents_total ) )
-		);
```

Comment:
Should we remove the `aria_label`?

Reply:
I'm not sure to understand your comment. This `aria-label` has been moved to the frontend script, as we no longer have the data in the PHP side:

https://github.com/woocommerce/woocommerce-blocks/blob/fix/9353-mini-cart-cache-friendly/assets/js/blocks/mini-cart/utils/data.ts#L27-L41

Reply:
Sorry, I wanted to cancel this comment